.github/workflows/ci.yml

```diff
@@ -49,7 +49,7 @@ jobs:
         run: yarn install --frozen-lockfile
 
       - name: 📦 Build & lint database functions
-        run: yarn build
+        run: yarn build:ci
```

Comment:
Heroku is trigger-happy on the project root's default `yarn build`, but we don't need Heroku to run that particular build command.